CHANGELOG.rst

```diff
@@ -1,3 +1,8 @@
+2.0.2
```

Comment:
Shouldn't we be at 2.0.1?

Also, the correct string here is `2.0.1 (unreleased)`.